readme.md

```diff
@@ -1,22 +1,24 @@
-# Jumpstart Live (JSL)
-_In person program for [Ada Developers Academy](http://adadevelopersacademy.org/) students who are not yet confident using loops, arrays, and hashes_
+# JumpStart Live (JSL)
+_In person program for [Ada Developers Academy](http://adadevelopersacademy.org/) students who are not yet well-versed using loops, arrays, and hashes_
 
 ## Overview
-Launched in January 2016, to support newly admitted Ada Developers Academy students who are not yet comfortable and confident using loops, arrays, and hashes. This JSL curriculum is a companion to the [Jumpstart curriculum](https://github.com/Ada-Developers-Academy/jump-start).
+JumpStart Live (JSL) was launched in January 2016 to support newly admitted Ada Developers Academy students who are not yet well-versed using loops, arrays, and hashes. This JSL curriculum is a companion to the [JumpStart curriculum](https://github.com/Ada-Developers-Academy/jump-start).
 
-JSL takes place in person at Ada 4 weeks prior to the start of the full-program. JSL reviews the most important content from the [Jumpstart curriculum](https://github.com/Ada-Developers-Academy/jump-start) and will not cover any new material.
+JSL takes place in person at Ada headquarters 4 weeks prior to the start of the full-program. JSL reviews the most important content from the [JumpStart curriculum](https://github.com/Ada-Developers-Academy/jump-start) and will not cover any new material.
 
 ## Class Structure
+* inverted classroom approach - students are expected to watch videos and
+complete exercises for the day before class
 * 2 hours, twice a week for 3 weeks
 * General class plan
 	* Welcome, goals, warm-up (25 mins)
-	* Lesson (25 mins)
+	* Follow up lesson (25 mins)
```

Comment:
Is this becoming "Follow up" lesson because its a follow up to their prep work? Or Follow up to what .. I think you can just keep this as Lesson unless you are trying to communicate something else here ..

Reply:
I agree, it makes sense to just keep it as Lesson. Updated.